### ISSUE TITLE
STK: GenFiles.py does **not** generate RACK-DATA.xsd

### ISSUE THREAD

**Issue by Ptival** [ge-high-assurance/RACK#303]
We encountered a bug with an updated `Add.py` where some classes would be missing in the CDR.

It turns out, while `Add.py` had been updated, `RACK-DATA.xsd` was a stale copy, and `createCDR` ignores anything that is not listed in the XSD.

The culprit seems to be `GenFiles.py` that did not regenerate `RACK-DATA.xsd`.

**Steps to reproduce:**

- Delete `Evidence/Add.py` and `Evidence/RACK-DATA.xsd`
- Run `GenFiles.py`
- `Evidence/Add.py` has been restored, `Evidence/RACK-DATA.xsd` is still missing.

**Comment by russell-d-e** [comment]
@Ptival was the `RACK-DATA.xsd` being generated somewhere else? 

I tried running locally and I was having the xsd generated correctly.

I noticed that when creating the xsd file I was was sloppy used a hard coded a path:
```
with open(r"..\Evidence\RACK-DATA.xsd","w") as xmlFile:`
```
instead of using the `os.path.join(os.path.split(sys.argv[0])[0]...` like I did for `Add.py`
```
EvidenceFilePath = os.path.join(os.path.split(sys.argv[0])[0],"..","Evidence","Add.py")
with open(EvidenceFilePath,"w") as pyFile:
```
I have updated my branch to correct this can you try this updated version and let me know if the xsd is being generated in the correct place

**Comment by Ptival** [comment]
Ah!

Indeed, the script was generating a file named `..\Evidence\RACK-DATA.xsd`, which is hidden by default on a Unix system since the name starts with a period.

And the fix on the `ScrapingToolKit` branch works for me, modifying the correct `RACK-DATA.xsd` file in the `Evidence` folder.

**Comment by Ptival** [comment]
Fixed by ce6f45d1998ba7f7e635c342fbc69b9bf841b9da
